### ISSUE TITLE
The npm install is installing the wrong version.

### ISSUE THREAD

**Issue by developerjohan01** [kavil/ng2SmoothScroll#6]
The npm package needs to be updated.
The `npm install ng2-smooth-scroll` installs version 1.0.1 and not 2.0.0.

**Comment by florestankorp** [comment]
Same. Any ideas how to install @2.0.0? Getting 
```
npm ERR! notarget No matching version found for ng2-smooth-scroll@2.0.0
```

**Comment by developerjohan01** [comment]
No, not if you want to use _this_ package with npm...  (You _could_ clone this project and publish your own component on npm with private or public scope... )
I ended up using a different and simpler solution without all the features of this component, which I didn't need.
